composer.json

```diff
@@ -16,10 +16,12 @@
         "guzzlehttp/guzzle": "^7.0.1",
         "intervention/image": "^2.4",
         "lab404/laravel-impersonate": "^1.2",
+        "laravel/cashier": "^13.6",
         "laravel/framework": "^8.0",
         "laravel/helpers": "^1.0",
         "laravel/tinker": "^2.0",
         "livewire/livewire": "^2.4",
+        "stripe/stripe-php": "^7.107",
```

Comment:
```suggestion
```
This dependency isn't required. [Cashier](https://github.com/laravel/cashier-stripe/blob/dcdd389f245a338854ca989cef300557d21f5025/composer.json#L31) already depends on it. So it will be required thanks to Cashier. No need to require it twice. The two versions might also cause a collision.